.readthedocs.yml

```diff
@@ -1,10 +1,22 @@
+version: 2
+
 build:
   image: latest
 
+# Install regular dependencies.
+# Then, install special pinning for RTD.
 python:
   version: 3.6
-  pip_install: true
-  extra_requirements: ['docs', 'all']
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
+        - all
+    - requirements: docs/rtd_requirements.txt
```

Comment:
I would maybe leave this config in the top directory, but it's just a slight preference.

Reply:
Ah, feel free to move it if you like. I should have checked where the old file was. My bad.

Reply:
I don't feel strongly enough about it to move now. If others think the same then sure.